packages/phone-number-privacy/test/salt-generation/query-quota.test.ts

```diff
@@ -0,0 +1,73 @@
+import { getPerformedQueryCount } from '../../src/database/wrappers/account'
+import { QueryQuota } from '../../src/salt-generation/query-quota'
+
+const ACCOUNT = '0x78dc5D2D739606d31509C31d654056A45185ECb6'
+const PHONE_NUMBER = '+1234567890'
+const queryQuota: QueryQuota = new QueryQuota()
+
+jest.mock('../../src/database/wrappers/account')
+const mockPerformedQueryCount = getPerformedQueryCount as jest.Mock
+
+// TODO (amyslawson) figure out how to change this mock for different unit tests
+// jest.mock('@celo/contractkit')
+// import { newKit } from '@celo/contractkit'
+// const mockNewKit = newKit as jest.Mock
+jest.mock('@celo/contractkit', () => {
+  const utils = require('../utils')
```

Comment:
why do you import utils here via require instead of just `import`ing it at the top?

Reply:
jest cannot find it otherwise...  apparently anything within the mock context needs to be defined there

Reply:
Hmm, maybe because of how jest implicitly hoists all mocks to the top of the file, except for variables that contain 'mock' in the name (IIRC)